homework-9/main.js

```diff
@@ -0,0 +1,68 @@
+// Bai 1: tinh thue va luong rong
+const getSalery = (luong) => {
+    if(luong >= 15000000){
+        console.log("Thuế thu nhập = ",luong*30/100);
+        console.log("Lương ròng = ",luong*70/100);
+    }
+    else if (luong < 15000000 && luong > 7000000) {
```

Comment:
Ngoặc đóng của block trước và mệnh đề sau nằm cùng 1 dòng.
Điều kiện cần rút ngắn gọn lại như bên dưới
```suggestion
    } else if (salary > 7000000) {
```